src/app/home/login/reset-pass.component.ts

```diff
@@ -14,30 +15,32 @@ import { MatDialogRef } from '@angular/material';
     </div>
     <div class="modal-body">
         <label for="email">E-mail</label>
-        <input id="email" type="email" required [(ngModel)]="email" email
-        #EMAIL="ngModel" placeholder="E-mail" class="form-control"/>
+        <input [formControl]= "email" id="email" type="email" required
+         placeholder="E-mail" class="form-control"/>
     </div>
     <div class="modal-footer">
-        <button class="btn btn-md btn-primary" [disabled]="!EMAIL.valid" (click)="resetPass()">Resetar Senha</button>
+        <button class="btn btn-md btn-primary" [disabled]="!email.valid" (click)="resetPass(email)">Resetar Senha</button>
     </div>
   `,
   providers: [NotificacaoService],
 })
 export class ResetPassComponent {
-  email: string = '';
+  email: FormControl;
 
   constructor(
     public dialogRef: MatDialogRef<any>,
     private authService: AuthService,
     private notificacao: NotificacaoService,
-  ) { }
+  ) {
+    this.email = new FormControl('', Validators.email);
+  }
 
   closeDialog() {
     this.dialogRef.close();
   }
 
   resetPass() {
```

Comment:
com isso faltou adicionar o parâmetro e-mail aqui
```suggestion
  resetPass(email: string) {
```